### ISSUE TITLE
Animated transitions in vis.js

### ISSUE THREAD

**Issue by bavoke1** [almende/vis#2758]
Does vis.js currently support animated transitions like D3.js does? I would like to transition certain properties of my network (node size, edge size, color ...) with a specified duration and/or delay.

**Comment by Tooa** [comment]
No, `visjs` does not have such a feature. However, we are open for pull requests. You might pick it up if you like to.

**Comment by rasmusblockzero** [comment]
I have an implementation that allow for animations of some opacity and size values. Though the actual selection of which values to animate are a bit hard coded for what I needed. It might be generalizable without too much work but if so there are some architectural discussions I'd need input on too. 

I don't think it had to be a breaking change @mojoaxel I believe it can be done in css-ish opt-in fashion

**Comment by wimrijnders** [comment]
@rasmusblockzero Please, feel free to create a PR for this!
